frontend/appflowy_flutter/packages/flowy_infra_ui/lib/src/flowy_overlay/appflowy_popover.dart

```diff
@@ -152,7 +153,7 @@ extension on BuildContext {
         side: BorderSide(
           width: 1,
           strokeAlign: BorderSide.strokeAlignOutside,
-          color: borderColor,
+          color: color != Colors.transparent ? borderColor : color!,
```

Comment:
@LucasXu0 This is for another issue, I noticed when setting the background of the Popover to transparent, it will still have a border (borderColor). I chose this workaround since I don't see a need to expand on it right now.